recipes/jags/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "JAGS" %}
+{% set version = "4.3.0" %}
+{% set sha256 = "8ac5dd57982bfd7d5f0ee384499d62f3e0bb35b5f1660feb368545f1186371fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://sourceforge.net/projects/mcmc-jags/files/JAGS/4.x/Source/{{ name }}-{{ version }}.tar.gz
```

Comment:
Could change the `4.x` to `{{ version[0] }}.x`, to make life slightly easier if a 5.x comes out.